src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: auto;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
+}
+
+.stars__star {
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star--noMargin {
+  margin-left: 0;
+}
+
+.star--noActive {
```

Comment:
better to use nth-of-type / nth-child pseudo-classes instead of class `star--noActive`/`star--active`:smiley: